action.yml

```diff
@@ -167,7 +170,7 @@ runs:
       shell: bash
       run: |
         echo ::group::Srtool build of chain ${{ inputs.chain }}
-        CMD="docker run -i --rm -e PACKAGE=${{ env.PACKAGE }} -e RUNTIME_DIR=${{ env.RUNTIME_DIR }} -v ${{ env.WORKDIR }}:/build ${{ env.SRTOOL_IMAGE }} build --app --json -cM"
+        CMD="docker run -i --rm -e PACKAGE=${{ env.PACKAGE }} -e RUNTIME_DIR=${{ env.RUNTIME_DIR }} -e BUILD_OPTS -v ${{ env.WORKDIR }}:/build ${{ env.SRTOOL_IMAGE }} build --app --json -cM"
```

Comment:
I would have preferred keeping it coherent here with either:
- `-e BUILD_OPTS=${{ env.BUILD_OPTS }}` like the rest
- or your admitedly more legible version everywhere 

I won't block the PR for that nonetheless as you went the safe route :)

Reply:
@chevdor `BUILD_OPTS` will have spaces and `=` so it will mess `CMD` when constructed. If I pass it as `-e BUILD_OPTS` it will read directly from env when running. I tried wrapping it with single quote but no luck. This was the only way I was able to get it running.